### PR TITLE
Node attribute for specifying orientations with quaternions.

### PR DESCRIPTION
**Pull request by hernando**
Clarified how multi-valued attributes can be included in node types
CSV files.

Removed the text about standarized morphology orientations as it isn't
explicitly stated anywhere what that means.

### CODE REVIEW

**Comment by hernando** [comment]
Do not merge yet, the examples need to be adapted. However I wanted to have a review before proceeding and changing all the examples (for which I need to modify bmtk and the generating scripts first).

**Comment by hernando** [comment]
Related to #57

**Comment by hernando** [comment]
I just changed the order of the quaternion elements to follow the most convention I've seen in different libraries.

**Comment by arsenius7** [comment]
LGTM

File: docs/SONATA_DEVELOPER_GUIDE.md [hunk 1]
```diff
@@ -549,13 +548,22 @@ For model_processing=*"fullaxon"*, the biophysical neuron will construct and sim
...
+
+**rotation_angle_zaxis [FLOAT], rotation_angle_yaxis[FLOAT], rotation_angle_xaxis[FLOAT]** - alternative representation of the rotation of the morphology around the soma.
 
-Each morphology is first moved from its original coordinates in SWC to such a location that the soma is at (0, 0, 0).  Then, three rotations are applied to each morphology, in exactly the following sequence: (1) rotation around the z-axis; (2) rotation around the y-axis; (3) rotation around the x-axis; all rotations are around the axes of the global coordinate system.  Then, each morphology is shifted to such a location that its soma is at (x, y, z) coordinates specified in the node/node_type files.  The angles for the three rotations are also provided in the nodes/node_types files, in radians.  If a column is not provided, it is assumed that the rotation angle around that axis is 0 (that is, no rotation around that axis is applied).
+This is an alternative representation to quaternions for cell orientation. Implementations are required to support both representations, but users are strongly encouraged to only use quaternions due to its non ambiguity.
```
[the remark this answers is not in the document]
Comment (hernando) [reply docs/SONATA_DEVELOPER_GUIDE.md]: I would say that a file should not contain both as it is implementation defined which one takes precedence.

**Comment by hernando** [comment]
@kaeldai If I don't get any comments from you by tomorrow I will merge this PR

File: docs/SONATA_DEVELOPER_GUIDE.md [hunk 2]
```diff
@@ -150,7 +150,7 @@ The format used is SWC ( [http://www.neuronland.org/NLMorphologyConverter/Morpho
 
 It is not required that the soma is located at 0,0,0 in the SWC file, but in cases where the morphology has a soma, the soma will be re-centered to 0,0,0 upon loading into the circuit.  Node translations will then be applied to this recentered morphology. This behavior can be overridden by the optional reserved attribute "recenter" for nodes and node_types.  See "Representing networks of neurons" for more details.
 
-It is recommended, but not required, that morphologies in a network have a standardized orientation, so that their orientation vectors in the network can be inferred from node rotation angles.  For circuits for which this is true, the circuit producer can declare it with the "standardized_morphology_orientation" entry in the circuit_config.json, so downstream users can safely assume it.  See "Tying it all together - the circuit_config file" for more details.
```
Comment (hernando) [inline docs/SONATA_DEVELOPER_GUIDE.md]: I removed the second sentence because it is not formalized what "standardized_morphology_orientation" means.

**Pull request merged**